### ISSUE TITLE
null Exception

### ISSUE THREAD

**Issue by Fshamri** [esafirm/Android-ReactiveGPlus#1]
i am getting null exception message after i click on login. 
i guess the problem is here : ViewObservable.clicks(button).subscribe(new Action1<OnClickEvent>()

**Comment by esafirm** [comment]
Have you setup your G+ integration yet? 
https://developers.google.com/+/mobile/android/getting-started

**Comment by Fshamri** [comment]
yes i have done the integration process with G+. i get null exception just after i accept the authorization request. 
 i have cloned your repo. and imported to my android studio and got the same problem.

BTW, i am taking your code as learning bais to RxAndroid. 
thanks for it
